specs/swagger.yml

```diff
@@ -2913,6 +2913,22 @@ paths:
           $ref: '#/responses/unauthorizedResponse'
         '429':
           $ref: '#/responses/rateLimitedResponse'
+  /database/checkDataIntegrity:
+    post:
+      summary: Runs an data integrity scan on the datastore and returns any found issues
+      description: Runs an data integrity scan on the datastore and returns any found issues. Requires admin access.
+      security:
+        - ztSession: []
+      tags:
+        - Database
+      operationId: checkDataIntegrity
+      parameters:
+        - $ref: '#/parameters/fixErrors'
```

Comment:
Seems odd to have a flag that fixes the error on the check. Seems odd to also have a `post` that is a read.

I'd suggest `POST /database/fix-data-integrity` and `GET /database/check-data-integrity`